app/routes/daily.tsx

```diff
@@ -475,8 +475,7 @@ function Card({
                 type="submit"
                 name="_action"
                 className="focus:shadow-outline m-1 h-[85px] w-[85px] rounded-lg bg-gray-700 font-bold
-                overflow-hidden whitespace-nowrap text-ellipsis
-          text-white transition-colors duration-150 hover:bg-gray-800"
+                overflow-hidden text-white transition-colors duration-150 hover:bg-gray-800"
```

Comment:
okay but what if the text is very long?

Reply:
Exercices labels containing spaces will go to the line until overflow. Single word text will just overflow and be hidden. It's a better solution than single line labels because it allows to differentiate the several labels that begin by the same word.

It could be great to also preview the full label in a popover when hovering on the exercice button.